dx-1005 added missing prefillInfo where msisdn is in Swish (#442)
SwedbankPay/developer.swedbankpay.com: payments/swish/direct.md

|                  | └─➔&nbsp;`productCategory`   | `string`      | A product category or number sent in from the payee/merchant. This is not validated by Swedbank Pay, but will be passed through the payment process and may be used in the settlement process.                                                                                                     |
|                  | └─➔&nbsp;`orderReference`    | `String(50)`  | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                                                                                                            |
|                  | └─➔&nbsp;`subsite`           | `String(40)`  | The subsite field can be used to perform split settlement on the payment. The subsites must be resolved with Swedbank Pay reconciliation before being used.                                                                                                                                        |

> |                  | └➔&nbsp;`prefillInfo`        | `object`  | An object that holds prefill information that can be inserted on the payment page.                                                                                                                                                                                                                 |

Reviewer: ```suggestion
|                  | └➔&nbsp;`prefillInfo`        | `object`  | An object holding information which, when available, will be prefilled on the payment page.                                                                                                                                                                                                                 |
```
